src/dash-playlist-loader.js

```diff
@@ -751,21 +748,23 @@ export default class DashPlaylistLoader extends EventTarget {
       }
     }
 
-    // if minimumUpdatePeriod is invalid or <= zero skip creating a timeout
+    // if minimumUpdatePeriod is invalid or <= zero, which
+    // can happen when a live video becomes VOD. skip timeout
+    // creation.
```

Comment:
I think we can use the DASH Interop as a reference:

https://dashif.org/docs/DASH-IF-IOP-v4.2-clean.htm

```
4.6.4.             Transition Phase between Live and On-Demand

In the scenario for which the same MPD URL is used for live and On-Demand content, once the URL and publish time of the last Segment is known for the live service, and the duration of the service is known as well, the service provider acts as defined in clause 4.4.3.1, i.e.,

-          adds the attribute MPD@mediaPresentationDuration
-          removes the attribute MPD@minimumUpdatePeriod
```

If the value is `< 0` we should probably issue a warning, as I don't think that should be valid.